_projects/Micromouse.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Micromouse
+description: Design a micromouse PCB from scratch and implement maze floodfill algorithms on it!
+importance: 1
+---
+
+| Project Domains                                                           | Mentors                      | Project Difficulty |
+|---------------------------------------------------------------------------|------------------------------|--------------------|
+| PCB Designing, Embedded C, Control Systems, Graph Algorithms, Electronics | Suraj Sonawane, Atharva Atre | Hard               |
+
+<br>
+
+### Project Description
+
+Develop a fastest micromouse maze-solving bot based on a PID controller that can find the shortest path in a given maze.
```

Comment:
Add some images and gifs for the project and make this more interesting, use GPT!!

Reply:
Okay I will add.